scipy/signal/tests/test_signaltools.py

```diff
@@ -33,70 +32,90 @@
 from scipy._lib import _testutils
 from scipy._lib._util import ComplexWarning, np_long, np_ulong
 
+from scipy._lib._array_api import  xp_assert_close, xp_assert_equal, is_cupy
+from scipy.conftest import array_api_compatible
 
-class _TestConvolve:
+pytestmark = [pytest.mark.usefixtures("skip_xp_backends")]
+skip_xp_backends = pytest.mark.skip_xp_backends
 
-    def test_basic(self):
-        a = [3, 4, 5, 6, 5, 4]
-        b = [1, 2, 3]
+
+class TestConvolve:
+
+    @skip_xp_backends("jax.numpy", reason="dtypes do not match")
```

Comment:
and below (or feel free to adjust the mark so that `reason` works...)

```suggestion
    @skip_xp_backends("jax.numpy", reasons=["dtypes do not match")]
```